src/components/MyHeader.jsx

```diff
@@ -0,0 +1,143 @@
+import React, { Component } from "react";
+import "bootstrap/dist/css/bootstrap.css";
+import TodoList from "./TodoList";
+import UUID from "uuid";
+
+class MyHeader extends Component {
+  state = {
+    tasks: [],
+    nextId: UUID(),
+    input: "",
+    editItem: false,
+    completed: [],
+    completedTask: ""
+  };
+  handleInputChange = e => {
+    this.setState({
+      ...this.state,
+      input: e.target.value
+    });
+  };
+
+  addTodo = e => {
+    e.preventDefault();
+    /**
+     * type task = {
+     *  id: UUID,
+     *  content: String,
+     *  completed: bool,
+     * }
+     */
+
+    if (this.state.input === "") {
+      alert("todo cannot be empty");
+    } else {
+      const addTask = {
+        id: this.state.nextId,
+        content: this.state.input
+      };
+
+      const updatedTasks = [...this.state.tasks, addTask];
+
+      this.setState(currentState => {
+        return {
+          ...currentState,
+          tasks: updatedTasks,
+          nextId: UUID(),
+          input: ""
+        };
+      });
+    }
+  };
+
+  nullWarning() {
+    let addWarning = "btn m-2 btn-";
+    addWarning += this.state.input === "" ? "warning" : "success";
+    return addWarning;
+  }
+
+  clearTasks = c => {
+    c.preventDefault();
+    this.setState({
+      tasks: []
+    });
+  };
+
+  deleteTask = id => {
+    const filterTask = this.state.tasks.filter(input => input.id !== id);
+    this.setState({
+      tasks: filterTask
+    });
+  };
+
+  editTask = id => {
+    const filterTask = this.state.tasks.filter(input => input.id !== id);
+    const selectedTask = this.state.tasks.find(input => input.id === id);
+    // console.log(selectedTask);
+    this.setState({
+      tasks: filterTask,
+      input: selectedTask.content,
+      id: id,
+      editItem: true
+    });
+    //console.log(this.state.input);
```

Comment:
commented console.logs should be removed

Reply:
@theAndrewCline ok cleared